### ISSUE TITLE
Disk IO stats missing after upgrade to 5.5.x kernel

### ISSUE THREAD

**Issue by sjclayton** [nicolargo/glances#1601]
#### Description

Disk IO stats are missing since upgrading to kernel 5.5.x, nothing else changed on system prior to noticing this..

Upon load the Disk IO which used to just show up doesn't anymore, I have no special config file set for glances and pressing 'd' to toggle doesn't do anything now.

Nothing relevant appears in glances log file.

#### Versions

* Glances & psutil:
Glances v3.1.3 with psutil v5.6.7

* Operating System:
Arch Linux x86_64
Kernel: 5.5.3-zen1-1-zen

**Comment by nicolargo** [comment]
Hi @sjclayton 

can you please copy/paste the result of the following command line:

```
$ python
>>> import psutil
>>> psutil.disk_io_counters(perdisk=True)
```

Thanks !

**Comment by sjclayton** [comment]
@nicolargo 

Here you go

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/site-packages/psutil/__init__.py", line 2168, in disk_io_counters
    rawdict = _psplatform.disk_io_counters(**kwargs)
  File "/usr/lib/python3.8/site-packages/psutil/_pslinux.py", line 1125, in disk_io_counters
    for entry in gen:
  File "/usr/lib/python3.8/site-packages/psutil/_pslinux.py", line 1098, in read_procfs
    raise ValueError("not sure how to interpret line %r" % line)
ValueError: not sure how to interpret line '   8       0 sda 116793 37246 5612669 62599 195785 106137 13318756 475261 0 95786 444535 0 0 0 0 18728 19348\n'
```

**Comment by sjclayton** [comment]
Appears to be fixed in psutil 5.7.0, closing this.
